### PR TITLE
Use correct Nassim fonts for Arabic, Pashto && Urdu services.

### LINKED ISSUE #2713: Use correct Nassim fonts for Arabic, Urdu, Pashto
**Testing notes**
**[Tester to complete with developer when issue is picked up]**

**Is your feature request related to a problem? Please describe.**
Following https://github.com/bbc/simorgh/issues/2732 we are temporarily going to use NassimPersian for all of APUP.

Once https://github.com/bbc/psammead/issues/735 is done all of the correct Nassim fonts will be available in Psammead, meaning that we can use the correct version of each font for the three remaining Arabic-script services (Arabic, Urdu, Pashto).

respective fonts for each of the services:
```
  F_NASSIM_ARABIC_REGULAR
  F_NASSIM_ARABIC_BOLD
  F_NASSIM_PASHTO_REGULAR
  F_NASSIM_PASHTO_BOLD
  F_NASSIM_URDU_REGULAR
  F_NASSIM_URDU_BOLD
```

**Describe the solution you'd like**
- Similar to how `persian` [does it](https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/persian.js) add the correct nassim fontfaces to each of the service configs for Arabic, Urdu and Pashto
- Add screenshots for before and after and get UX to review
- Check if any changes are required to Storybook in SImorgh

**Testing notes**
- Check using DevTools that the correct version of the Nassim font is used on each service's articles
- Check that no other styling is adversely impacted (e.g. inline link changes)
- Test how it looks on older/smaller devices
- Dev and test to decide on best way of providing test coverage for correct font being used

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by RayNjeri**
Signed-off-by: Rachael Njeri <rachael.njeri@andela.com>

Resolves #2713

**Overall change:** _Pulls in the correct Nassim fonts for Urdu, Pashto and Arabic services._

**Code changes:**

- _Pulled in Correct fonts into the respective config files._
- _Bumped Psammead packages._
- _Updated Snapshots._

Testing: 
This requires a full simorgh regression due to the vast number of psammead packages bumped

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by jamesdonoh** [comment]
Blocked on bbc/psammead#1592

**Comment by jimjohnsonrollings** [comment]
Blocking PR is now merged

File: package.json [hunk 1]
```diff
@@ -78,28 +78,28 @@
...
-    "@bbc/psammead-styles": "^1.2.0",
+    "@bbc/psammead-paragraph": "^2.1.2",
+    "@bbc/psammead-section-label": "^2.1.3",
+    "@bbc/psammead-sitewide-links": "^3.0.1",
+    "@bbc/psammead-story-promo": "^2.3.1",
+    "@bbc/psammead-story-promo-list": "^1.1.2",
+    "@bbc/psammead-styles": "^2.0.1",
     "@bbc/psammead-timestamp-container": "^2.3.2",
-    "@bbc/psammead-visually-hidden-text": "^1.1.0",
+    "@bbc/psammead-visually-hidden-text": "^1.1.1",
```
Comment (jamesdonoh) [inline package.json]: Why are all these version numbers being updated? I thought all the bumping had been taken care of separately.

Comment (RayNjeri) [reply package.json]: In psammead but not in Simorgh , if not updated we will have duplicates issue, 
we need to use updated packages that have psammead-styles @2.0.1 as a  dependency which has the new Nassim fonts

**Review by jamesdonoh — APPROVED** [review]

**Review by 12 — CHANGES_REQUESTED** [review]
In Firefox, I am not getting Nassim, and all lazyloaded images are not working...

File: src/app/components/ErrorPage/index.test.jsx [hunk 2]
```diff
@@ -25,4 +25,19 @@ describe('ErrorMain', () => {
     'should correctly render for an error page for Persian',
     <ErrorMain {...messaging} script={arabic} service="persian" />,
   );
+
```
Comment (EinsteinNjoroge) [inline src/app/components/ErrorPage/index.test.jsx]: consider simplifying like this 

  ```
const arabicServices = ['persian', 'arabic', 'pashto', 'urdu'];
  arabicServices.forEach(service => {
    shouldMatchSnapshot(
      `should correctly render for an error page for ${service}`,
      <ErrorMain {...messaging} script={arabic} service={service} />,
    );
  });
```

**Review by 12 — APPROVED** [review]

**Comment by dr3** [comment]
This requires a FULL simorgh regression due to the vast number of psammead packages bumped

**Comment by RayNjeri** [comment]
> In Firefox, I am not getting Nassim, and all lazyloaded images are not working...

Resolved

**Review by sareh — APPROVED** [review]
👍 Looks great. 
BBC Nassim Arabic Regular and Bold loading correctly:
Screen Shot 2019-08-08 at 14 59 36
Screen Shot 2019-08-08 at 14 59 19

Also checked the other service pages and they're working well, too. 🎉

File: scripts/bundleSize.sh [hunk 3]
```diff
@@ -5,8 +5,8 @@
 # If updating:
 # - The min value should be 5K less than the smallest service bundle and max should be 5K greater than the largest bundle
 # - Bundle sizes can be viewed by running a production build
-min=494
-max=522
+min=502
+max=535
```
Comment (dr3) [inline scripts/bundleSize.sh]: Howcome this is going up so much?

**Review by dr3 — CHANGES_REQUESTED** [review]

File: scripts/bundleSize.sh (hunk 3 above)
Comment (dr3) [reply scripts/bundleSize.sh]: This adds 5k to all services except news & persian, which it adds 10/11. Howcome?

Comment (dr3) [reply scripts/bundleSize.sh]: Found it, https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/fonts.js gained 10k and we dont tree shake it :(

**Review by dr3 — APPROVED** [review]
Shame 10k increase, my bad, this is fine to continue, ile make a PR to free shake psammead styles and pull it in later

**Comment by sareh** [comment]
Going to use this PR to update to all the latest dependencies: https://github.com/bbc/simorgh/pull/3030 
This is so that any issues that arise won't be bundled in with this font work.

**Comment by david-boydell** [comment]
Front pages regression looks good.

**Comment by jamesbrumpton** [comment]
We were having issues testing this on `local` due to ngrok/BrowserStack conflicts. However, when this was deployed to Test it was working as expected. Based on this, I'm happy for this to be merged.

**Pull request merged**